views/pages/guides/runtime-environments.hbs

```diff
@@ -80,22 +80,19 @@
             </h3>
 
             <p>
-                The following code snippet will help you polyfill the Node.js runtime when the {{code "Intl"}} APIs are missing, or if the built-in {{code "Intl"}} is missing locale data that's needed for your app:
+                The following code snippet uses the <a href="https://github.com/andyearnshaw/Intl.js">{{code "intl"}}</a> and <a href="https://github.com/yahoo/intl-locales-supported">{{code "intl-locales-supported"}}</a> npm packages which will help you polyfill the Node.js runtime when the {{code "Intl"}} APIs are missing, or if the built-in {{code "Intl"}} is missing locale data that's needed for your app:
             </p>
 
 {{#code "js"}}
+var areIntlLocalesSupported = require('intl-locales-supported');
```

Comment:
Can we explain here why this is necessary? Even though it's explained in `note-intl-node.hbs` it might be a surprise for some people.